src/components/Card/Card.vue

```diff
@@ -1,28 +1,43 @@
 <template>
   <article class="location-card" :id="id">
-    <div class="button-container">
-      <button class="button button-refresh" type="button" name="button" v-on:click.prevent="refreshLocation">🔄</button>
-      <button class="button" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    <div v-if="currentLocation">
+      <p>Current Location</p>
     </div>
-    <h2>{{ locale }}</h2>
+    <div v-else class="button-delete-container">
+      <button class="card-button button-delete" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    </div>
+    <h2>{{ formatLocation() }}</h2>
     <img :src="require(`../../assets/${formatWeatherIcon()}.png`)" :alt="weatherIconAlt" class="weather-icon">
     <h3>Weather</h3>
     <p>{{ temperature }}°F</p>
     <p>Wind Speed: {{ windSpeed }}mph</p>
     <p>Humidity: {{ humidity }}%</p>
+    <hr>
     <h3>Air Quality</h3>
-    <p>{{ aqi }} AQI</p>
-    <p>{{ determineAqiMessage() }}</p>
-    <p>{{ timeStamp }}</p>
+    <div class="aqi-message">
+      <div class="aqi-circle" :style="determineAqiStyle()"><p>AQI: {{ aqi }}</p></div>
+      <p class="aqi-rating"><b>{{ determineAqiRating() }} </b><a href="https://www.airnow.gov/aqi-and-health/" target="_blank"><img src="https://img.icons8.com/metro/16/000000/info.png"/></a></p>
```

Comment:
I think it would be a little more readable if we were able to break this onto multiple lines. I'm not sure if that would affect the styling though so it's totally fine if we're not able to do so.

Reply:
Agreed, I'll make a note of this in Trello!